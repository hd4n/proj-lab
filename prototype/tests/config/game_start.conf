
@fields
em e1
em e2
sh s1

@neighbors
e1 s1
e2 s1

@virologists
v1 e1
v2 e2