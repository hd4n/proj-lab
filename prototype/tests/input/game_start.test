load game_start.conf

getstats v1
getstats v2

getstats map