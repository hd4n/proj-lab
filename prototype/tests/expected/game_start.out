Load: sikeres
citizens.Virologist@7c16905e
citizens.Virologist@2a2d45ba
stat map:
	710714889 - 551734240
	650023597 - 551734240
	551734240 - 710714889
	551734240 - 650023597
